api/jobset/v1alpha1/jobset_webhook.go

```diff
@@ -73,6 +74,12 @@ func (js *JobSet) ValidateCreate() error {
 	var allErrs []error
 	// Validate that replicatedJobs listed in success policy are part of this JobSet.
 	validReplicatedJobs := replicatedJobNamesFromSpec(js)
+	for _, rjob := range js.Spec.ReplicatedJobs {
+		sum := int64(*(rjob.Template.Spec.Parallelism)) + int64(rjob.Replicas)
```

Comment:
A couple of things:
1) multiply not sum (replicas * parallelism < maxint)
2) check if rjob.Template.Spec.Parallelism is nil, in which case it parallelism is 1

Reply:
perhaps, you meant (replicas * parallelism > maxint)?